### PR TITLE
Add option to preset key in the ethereum keystore

### PR DESCRIPTION
**Pull request by svetomir**
This PR allows pre-setting of a key (and appropriate password) in the ethereum data dir.

If Values.key and Values.password are specified, (by default) they will be saved in following files, respectively:
/root/.ethereum/keystore/key.json
/root/.ethereum/password

Both, key and password are kept in a Kubernetes secrets.

### CODE REVIEW

File: eth-geth/templates/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+{{ if .Values.key }}
```
Comment (leovigna) [inline eth-geth/templates/secret.yaml]: Shouldn't this be `Values.password` and vice-versa? Or if both have to be specified a single conditional with AND operator would be clearer.

File: eth-geth/templates/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+kind: Secret
+metadata:
+  name: {{ include "geth.fullname" . }}-password
+stringData:
+  password: |
+{{ .Values.password | indent 4 }}
+{{- end }}
+
+---
+{{ if .Values.password }}
```
Comment (leovigna) [inline eth-geth/templates/secret.yaml]: See above comment

**Review by leovigna — CHANGES_REQUESTED** [review]
See comments regarding conditional statements on `secret.yaml` 

Can you also update the README file to add your options to the table?

File: eth-geth/templates/secret.yaml (hunk 1 above)
Comment (svetomir) [reply eth-geth/templates/secret.yaml]: You're right, my bad. Thank you

**Comment by svetomir** [comment]
> See comments regarding conditional statements on `secret.yaml`
> 
> Can you also update the README file to add your options to the table?

Thank you for the review. I pushed the changes.

**Comment by leovigna** [comment]
lgtm. I will soon merge this to the master branch and deploy it to the chart repository.

**Pull request merged**